### DISCUSSION TITLE
[BUG] Treeview plugin activating dynamically by jQuery not working

### DISCUSSION THREAD

**Discussion by CodeEaseWithAnu** [ColorlibHQ/AdminLTE#3215] [Q&A]
AdminLte 3 Tree view plugin activate dynamically by jQuery not working

I have seen treeview plug in of admin lte with ul list implement dynamically by following code

$('ul').Treeview(options) 
but its not working
i have load all required js and css file, please suggest me how dynamically tree view plugin of adminlte activate through jQuery
is there any initialization required or any extra js file nedded

**Comment by REJack** [comment]
Did you tried this with AdminLTE v3.0.5 or the latest rc release?
